backend_addon/{{ cookiecutter.__folder_name }}/Makefile

```diff
@@ -62,7 +62,7 @@ config: instance/etc/zope.ini
 build-dev: config ## Install Plone packages
 	@echo "$(GREEN)==> Setup Build$(RESET)"
 	$(BIN_FOLDER)/pipx run mxdev -c mx.ini
-	$(BIN_FOLDER)/uv pip install -r requirements-mxdev.txt
+	VIRTUAL_ENV=.venv $(BIN_FOLDER)/uv pip install -r requirements-mxdev.txt
```

Comment:
uv is a standalone binary installed with pip (to have installed in the pip bin directory), we install it in a venv but it does not find it as the first default. Using VIRTUAL_ENV seems the only option (other than activate the venv) so it is ok.